mods/ts/rules/defaults.yaml

```diff
@@ -18,69 +18,80 @@
 ^GainsExperience:
 	GainsExperience:
 		Upgrades:
-			500: rank, firepower, damage, speed, reload
-			1000: rank, firepower, damage, speed, reload, selfheal, eliteweapon
-	FirepowerMultiplier@EXPERIENCE:
-		UpgradeTypes: firepower
-		Modifier: 110, 130
-	DamageMultiplier@EXPERIENCE:
-		UpgradeTypes: damage
-		Modifier: 90, 75
-	SpeedMultiplier@EXPERIENCE:
-		UpgradeTypes: speed
-		Modifier: 120, 140
-	ReloadDelayMultiplier@EXPERIENCE:
-		UpgradeTypes: reload
-		Modifier: 90, 75
+			500: rank-veteran
+			1000: rank-elite
+	FirepowerMultiplier@VETERAN:
+		RequiresCondition: (rank-veteran && !rank-elite) || crate-firepower
```

Comment:
While I'm aware that this is the sole usecase for combined expressions, could this be changed now to implement the TS crate multipliers - https://github.com/LipkeGu/TSIniFiles/blob/master/rules.ini#L10467 and onwards with the last coloumn being them?